{
  "uuid": "3e548dfc-de27-4a6e-acae-e4dd33bedada",
  "historyId": "011acdc9dda5fd570f1af43701ab7619",
  "testCaseId": "5acb2df2272bfa3c0338b47aaa8372a2",
  "fullName": "OdinUiTests:OdinUiTests.Tests.Activity.TestActivityStatusWithOneAttemptTests.CheckStatusIsPendingReviewForAttemptsNeedsReview(System.String)",
  "labels": [
    {
      "name": "thread",
      "value": "NUnit.Fw.ParallelWorker#4"
    },
    {
      "name": "host",
      "value": "7631391289e3"
    },
    {
      "name": "language",
      "value": "C#"
    },
    {
      "name": "framework",
      "value": "NUnit 3"
    },
    {
      "name": "package",
      "value": "OdinUiTests.Tests.Activity"
    },
    {
      "name": "testMethod",
      "value": "CheckStatusIsPendingReviewForAttemptsNeedsReview"
    },
    {
      "name": "testClass",
      "value": "TestActivityStatusWithOneAttemptTests"
    },
    {
      "name": "parentSuite",
      "value": "Активность"
    },
    {
      "name": "suite",
      "value": "Контрольная"
    },
    {
      "name": "subSuite",
      "value": "Прохождение теста"
    },
    {
      "name": "severity",
      "value": "critical"
    },
    {
      "name": "tag",
      "value": "Активность"
    },
    {
      "name": "tag",
      "value": "Дежурство"
    },
    {
      "name": "tag",
      "value": "Регресс"
    }
  ],
  "links": [],
  "name": "Статус Ожидает проверки, если в попытке есть хотя бы один вопрос, требующий проверки в индивидуальной контрольной",
  "status": "broken",
  "statusDetails": {
    "known": false,
    "muted": false,
    "flaky": false,
    "message": "System.Reflection.TargetInvocationException : Exception has been thrown by the target of an invocation.\n  ----> OpenQA.Selenium.WebDriverTimeoutException : Timed out after 30 seconds\n  ----> OpenQA.Selenium.NoSuchElementException : no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[text()='Перейти к тесту']\"}\n  (Session info: chrome=132.0.6834.83); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception",
    "trace": "   at System.Reflection.MethodBaseInvoker.InvokeWithManyArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)\n   at Allure.Net.Commons.Steps.AllureStepAspectBase.Around(String name, Object[] args, Func`2 target, MethodBase metadata, Type returnType)\n   at OdinUiTests.Pages.ActivityCommonPage.__a$_around_ClickOnGotoTestButton_100666061_w_0(Object[])\n   at OdinUiTests.Pages.ActivityCommonPage.ClickOnGotoTestButton()\n   at OdinUiTests.Tests.Activity.TestActivityStatusWithOneAttemptTests.CheckStatusIsPendingReviewForAttemptsNeedsReview(String actId) in /azp/_work/2/s/Tests/Activity/TestActivityTests.cs:line 626\n   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)\n   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)\n--WebDriverTimeoutException\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)\n   at OdinUiTests.Pages.ActivityCommonPage.__a$_around_ClickOnGotoTestButton_100666061_o() in /azp/_work/2/s/Pages/ActivityCommonPage.cs:line 314\n   at OdinUiTests.Pages.ActivityCommonPage.__a$_around_ClickOnGotoTestButton_100666061_u(Object[])\n   at Allure.Net.Commons.Steps.AllureAbstractStepAspect.WrapSync[T](Func`2 target, Object[] args, MethodBase metadata, String stepName, List`1 stepParameters)\n   at InvokeStub_AllureAbstractStepAspect.WrapSync(Object, Span`1)\n   at System.Reflection.MethodBaseInvoker.InvokeWithManyArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)\n--NoSuchElementException\n   at OpenQA.Selenium.WebDriver.UnpackAndThrowOnError(Response errorResponse, String commandToExecute)\n   at OpenQA.Selenium.WebDriver.ExecuteAsync(String driverCommandToExecute, Dictionary`2 parameters)\n   at SeleniumExtras.WaitHelpers.ExpectedConditions.<>c__DisplayClass19_0.<ElementToBeClickable>b__0(IWebDriver driver)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)"
  },
  "stage": "finished",
  "steps": [
    {
      "name": "Открыть страницу активности",
      "status": "passed",
      "statusDetails": {
        "known": false,
        "muted": false,
        "flaky": false
      },
      "stage": "finished",
      "steps": [],
      "attachments": [],
      "parameters": [
        {
          "name": "activityId",
          "value": "\"17996\"",
          "excluded": false
        }
      ],
      "start": 1740539782725,
      "stop": 1740539783141
    },
    {
      "name": "Нажать на кнопку 'Перейти к тесту' со страницы активности",
      "status": "broken",
      "statusDetails": {
        "known": false,
        "muted": false,
        "flaky": false,
        "message": "Timed out after 30 seconds",
        "trace": "OpenQA.Selenium.WebDriverTimeoutException: Timed out after 30 seconds\n ---> OpenQA.Selenium.NoSuchElementException: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[text()='Перейти к тесту']\"}\n  (Session info: chrome=132.0.6834.83); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\n   at OpenQA.Selenium.WebDriver.UnpackAndThrowOnError(Response errorResponse, String commandToExecute)\n   at OpenQA.Selenium.WebDriver.ExecuteAsync(String driverCommandToExecute, Dictionary`2 parameters)\n   at SeleniumExtras.WaitHelpers.ExpectedConditions.<>c__DisplayClass19_0.<ElementToBeClickable>b__0(IWebDriver driver)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.ThrowTimeoutException(String exceptionMessage, Exception lastException)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition, CancellationToken token)\n   at OpenQA.Selenium.Support.UI.DefaultWait`1.Until[TResult](Func`2 condition)\n   at OdinUiTests.Pages.ActivityCommonPage.__a$_around_ClickOnGotoTestButton_100666061_o() in /azp/_work/2/s/Pages/ActivityCommonPage.cs:line 314\n   at OdinUiTests.Pages.ActivityCommonPage.__a$_around_ClickOnGotoTestButton_100666061_u(Object[])\n   at Allure.Net.Commons.Steps.AllureAbstractStepAspect.WrapSync[T](Func`2 target, Object[] args, MethodBase metadata, String stepName, List`1 stepParameters)"
      },
      "stage": "finished",
      "steps": [],
      "attachments": [],
      "parameters": [],
      "start": 1740539783141,
      "stop": 1740539815431
    }
  ],
  "attachments": [
    {
      "name": "CheckStatusIsPendingReviewForAttemptsNeedsReview.jpg",
      "source": "14aba7c2-6141-4c4e-a730-7aeaa7ed5064-attachment.jpg",
      "type": "image/jpeg"
    },
    {
      "name": "Console Output",
      "source": "76ddef87-6c1e-4bf2-aaef-fe04c49bdd59-attachment.txt",
      "type": "text/plain"
    }
  ],
  "parameters": [
    {
      "name": "actId",
      "value": "\"17996\"",
      "excluded": false
    }
  ],
  "start": 1740539782121,
  "stop": 1740539816011
}